chromium_src/chrome/common/channel_info_posix.cc

```diff
@@ -17,9 +18,31 @@ std::string GetChannelName() {
   return modifier;
 }
 
+#if defined(OS_LINUX)
+std::string GetDesktopName(base::Environment* env) {
+#if defined(OFFICIAL_BUILD)
```

Comment:
I'm confused, shouldn't there also be code removed somewhere for this? It was moved, right?

Reply:
I found it https://github.com/brave/brave-core/pull/4043/commits/f5fb5f52c56823fd19e07ee3cea51d393a7d607c#diff-3f00b2c227be3ff2ee4155dfccda2199L8

Reply:
so this is fine